packages/eslint-config-airbnb/README.md

```diff
@@ -26,6 +24,9 @@ If you use yarn, run `npm info "eslint-config-airbnb@latest" peerDependencies` t
   npx install-peerdeps --dev eslint-config-airbnb
   ```
 
+  If using **Yarn**, you can also use the shortcut described above if you have npm 5+ installed on your machine, as the command will detect that you are using Yarn and will act accordingly.
+  Otherwise, run `npm info "eslint-config-airbnb@latest"   peerDependencies` to list the peer dependencies and versions, then run `yarn add --dev <dependency>@<version>` for each listed peer dependency.
```

Comment:
```suggestion
  Otherwise, run `npm info "eslint-config-airbnb@latest" peerDependencies` to list the peer dependencies and versions, then run `yarn add --dev <dependency>@<version>` for each listed peer dependency.
```